zapcore/buffered_write_syncer_bench_test.go

```diff
@@ -30,7 +30,7 @@ import (
 
 func BenchmarkBufferedWriteSyncer(b *testing.B) {
 	b.Run("write file with buffer", func(b *testing.B) {
-		file, err := os.CreateTemp("", "log")
+		file, err := os.CreateTemp("", "test.log")
```

Comment:
Why doesn't this use b.TempDir()?

```suggestion
		file, err := os.CreateTemp(b.TempDir(), "test.log")
```